packages/project-build/src/shared/pages-utils.ts

```diff
@@ -1,6 +1,7 @@
 import type { Pages, Page, Folder } from "@webstudio-is/sdk";
 import { nanoid } from "nanoid";
 
+// @todo path needs to support folders now
```

Comment:
to do before merging

Reply:
So not ready for review?

Reply:
will send it with the next PR, turns there is a bunch of places that rely on page.path and now its always the entire ancestory + page.path